repondeur/zam_repondeur/models/amendement.py

```diff
@@ -57,7 +57,9 @@ class Amendement(Base):  # type: ignore
     # Relations.
     parent_pk = Column(Integer, ForeignKey("amendements.pk"), nullable=True)
     parent_rectif = Column(Integer, nullable=True)
-    parent = relationship("Amendement", uselist=False, primaryjoin=(parent_pk == pk))
+    parent = relationship(
+        "Amendement", uselist=False, remote_side=[pk], backref=backref("children")
```

Comment:
TODO(self): check if that change doesn't increase the number of queries to display the list of amendements and the visionneuse, even better: set a test for that 😇